Core/include/Acts/Utilities/GridBinFinder.hpp

```diff
@@ -0,0 +1,114 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
```

Comment:
Let's also update the copyright everywhere
```suggestion
// Copyright (C) 2024 CERN for the benefit of the Acts project
```